support/cas-server-support-ldap-monitor/src/main/java/org/apereo/cas/monitor/config/LdapMonitorConfiguration.java

```diff
@@ -36,16 +40,33 @@
     private CasConfigurationProperties casProperties;
 
     @Bean
+    ListFactoryBean pooledLdapConnectionFactoryHealthIndicatorFactoryBean() {
```

Comment:
Does the bean need to be public? If so, please mark it as so. Or more generally, specify the access keyword to be consistent and transparent.

Reply:
Beans MUST be public, otherwise Spring will be unable to access it. This is a accidental mistake.